pyleecan/Methods/Elmer/ElmerResultsVTU/get_meshsolution.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+from numpy import arange
+from meshio import read
+from os.path import join, split, splitext
+
+from SciDataTool import DataTime, Data1D, VectorField
+
+from ....Classes.SolutionData import SolutionData
+from ....Classes.SolutionVector import SolutionVector
+from ....Classes.MeshSolution import MeshSolution
+from ....Classes.MeshVTK import MeshVTK
+
+from ....Methods.Elmer.ElmerResultsVTU import ElmerResultsVTUError
+
+
+# TODO add groups, see get_meshsolution of MagFEMM
+
+
+def get_meshsolution(self):
```

Comment:
To be more coherent with MagFEMM, I would rename as "build_meshsolution" instead.